go.mod

```diff
@@ -45,7 +45,7 @@ require (
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
 	github.com/tendermint/go-amino v0.16.0
-	github.com/tendermint/tendermint v0.34.0-rc5
+	github.com/tendermint/tendermint v0.34.0-rc5.0.20201019080751-6f908eb81470
```

Comment:
shouldn't we use `@` syntax (with a commit)?

Reply:
as in replace in the go.mod? it is this way because the needed changes are not tagged yet.

Reply:
Ah, sorry. It's good. this is what you will get if you will do `go get package@commit` in a module. It will go to the sum file.